PSL_catalog.json

```diff
@@ -0,0 +1,156 @@
+{
+    "project_one_line": {
+        "start_header": null,
+        "end_header": null,
+        "source": "https://github.com/open-source-economics/OG-USA",
+        "type": "html",
+        "data": "<p>Dynamic Tax Scoring Model</p>"
```

Comment:
Maybe for data here:
`<p>Overlapping-Generations Model of the United States</p>`

Maybe want to say something about how its focus is fiscal policy analysis, but not sure if that's too long:
`<p>Overlapping-Generations Model for Evaluating Fiscal Policy in the United States</p>`

cc @rickecon